### DISCUSSION TITLE
[@types/mockjs] random.range type error

### DISCUSSION THREAD

**Discussion by bastarder** [DefinitelyTyped/DefinitelyTyped#58151] [Issues with a @types package]
`MockjsRandomBasic.range` type is `range(start?: N, stop?: N, step?: N): N;` now,  maybe `range(start?: N, stop?: N, step?: N): N[];` is correct；
[see](https://github.com/nuysoft/Mock/blob/refactoring/src/mock/random/basic.js#L102)

[a comment by an automated account is not shown]

**Comment by ChenKS12138** [comment]
> `MockjsRandomBasic.range` type is `range(start?: N, stop?: N, step?: N): N;` now, maybe `range(start?: N, stop?: N, step?: N): N[];` is correct； [see](https://github.com/nuysoft/Mock/blob/refactoring/src/mock/random/basic.js#L102)

Thanks, It seems right. PR is needed.

**Reply by bastarder** [reply]
> > `MockjsRandomBasic.range` type is `range(start?: N, stop?: N, step?: N): N;` now, maybe `range(start?: N, stop?: N, step?: N): N[];` is correct； [see](https://github.com/nuysoft/Mock/blob/refactoring/src/mock/random/basic.js#L102)
> 
> Thanks, It seems right. PR is needed.

i have fix this, can you help me to review. [#58152](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58152)

**Reply by ChenKS12138** [reply]
Review is done.
